stable/rethinkdb/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: rethinkdb
+description: The open-source database for the realtime web
+version: 0.0.1
+keywords:
+- rethinkdb
+- database
+- nosql
+home: https://www.rethinkdb.com
```

Comment:
[Which one do you suggest?](https://www.google.com/search?newwindow=1&site=&tbm=isch&source=hp&biw=2560&bih=1316&q=rethinkdb+logo&oq=rethinkdb+logo&gs_l=img.3..35i39k1.1077.2762.0.2925.15.15.0.0.0.0.136.1063.12j2.14.0....0...1.1.64.img..1.14.1056.0..0j0i24k1.IpcYB2a9CwM). I suppose the one with the "R" in a square?